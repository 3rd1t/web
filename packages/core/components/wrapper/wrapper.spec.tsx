import React from "react"
import { render } from "@testing-library/react"

import Wrapper from "./wrapper"

describe("Wrapper", () => {
  it("should render successfully", () => {
    const { baseElement } = render(
      <Wrapper>
        <h1>Hello World</h1>
      </Wrapper>,
    )
    expect(baseElement).toMatchSnapshot()
  })
})
