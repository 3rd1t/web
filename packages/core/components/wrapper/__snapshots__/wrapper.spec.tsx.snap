// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wrapper should render successfully 1`] = `
<body>
  <div>
    <div>
      <div
        class="pt-16 bg-gray-100"
      >
        <div
          class="fixed inset-x-0 top-0 z-20 bg-gray-100"
        >
          <nav
            class="container flex flex-wrap items-center justify-between h-16 px-4 mx-auto md:flex-no-wrap"
            role="navigation"
          >
            <a
              href="/"
            >
              <div
                class="flex items-center space-x-2"
              >
                <svg
                  class="h-6 text-gray-900 stroke-current"
                  fill="none"
                  viewBox="0 0 194 148"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M9 106.208L97.3883 17.8199L134.158 54.5894M185.07 41.8615L96.6814 130.25L59.9118 93.4803"
                    stroke-width="25"
                  />
                </svg>
                <div
                  class="text-xl font-semibold"
                >
                  <span
                    class="text-gray-800"
                  >
                    PERFOLIO
                  </span>
                </div>
              </div>
            </a>
            <div
              class="hidden md:block"
            >
              <div
                class="w-auto"
              >
                <ul
                  class="flex items-center space-x-10"
                >
                  <li>
                    <a
                      class="text-lg font-medium text-gray-700 transition duration-150 ease-in-out hover:text-black"
                      href="/"
                    >
                      Product
                    </a>
                  </li>
                  <li>
                    <a
                      class="text-lg font-medium text-gray-700 transition duration-150 ease-in-out hover:text-black"
                      href="/#features"
                    >
                      Features
                    </a>
                  </li>
                  <li>
                    <a
                      class="text-lg font-medium text-gray-700 transition duration-150 ease-in-out hover:text-black"
                      href="/#team"
                    >
                      Team
                    </a>
                  </li>
                  <li>
                    <a
                      class="text-lg font-medium text-gray-700 transition duration-150 ease-in-out hover:text-black"
                      href="/docs"
                    >
                      Documentation
                    </a>
                  </li>
                </ul>
              </div>
            </div>
            <div
              class="block md:hidden"
            >
              <div>
                <button
                  aria-label="open menu"
                  class="p-2 text-gray-700 hover:text-gray-900 focus:outline-none"
                  data-testid="toggle"
                >
                  <svg
                    class="w-6 h-6"
                    fill="none"
                    stroke="currentColor"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M4 6h16M4 12h16M4 18h16"
                      stroke-linecap="round"
                      stroke-linejoin="round"
                      stroke-width="2"
                    />
                  </svg>
                </button>
              </div>
            </div>
          </nav>
        </div>
        <h1>
          Hello World
        </h1>
      </div>
      <footer
        class="bg-carbon-900"
      >
        <div
          class="container mx-auto"
        >
          <div
            class="flex flex-wrap justify-between p-10  text-gray-100"
          >
            <div
              class="p-2 md:space-y-2"
            >
              <span
                class="text-xs font-semibold text-white uppercase"
              >
                Projects
              </span>
              <a
                class="block focus:outline-none text-gray-100 hover:text-white"
                href="https://perfol.io"
              >
                Home
              </a>
              <div
                class="flex items-center space-x-2"
              >
                <a
                  class="block focus:outline-none text-gray-100 hover:text-white"
                  href="https://analytics.perfol.io"
                >
                  Analytics
                </a>
                <span
                  class="px-1 text-sm bg-purple-800 rounded-sm"
                >
                  WIP
                </span>
              </div>
              <a
                class="block focus:outline-none text-gray-100 hover:text-white"
                href="https://research.perfol.io"
              >
                Research
              </a>
            </div>
            <div
              class="p-2 md:space-y-2"
            >
              <span
                class="text-xs font-semibold text-white uppercase"
              >
                Resources
              </span>
              <a
                class="block focus:outline-none text-gray-100 hover:text-white"
                href="/docs"
              >
                Documentation
              </a>
              <a
                class="block focus:outline-none text-gray-100 hover:text-white"
                href="perfol.io/roadmap"
              >
                Roadmap
              </a>
              <a
                class="block focus:outline-none text-gray-100 hover:text-white"
                href="/imprint"
              >
                Imprint
              </a>
              <a
                class="block focus:outline-none text-gray-100 hover:text-white"
                href="/privacy"
              >
                Privacy
              </a>
            </div>
            <div
              class="p-2 md:space-y-2"
            >
              <span
                class="text-xs font-semibold text-white uppercase"
              >
                Contact us
              </span>
              <a
                class="block focus:outline-none text-gray-100 hover:text-white"
                href="mailto:info@perfol.io"
              >
                contact@perfol.io
              </a>
            </div>
            <div
              class="p-2 md:space-y-2"
            >
              <span
                class="text-xs font-semibold text-white uppercase"
              >
                Source
              </span>
              <a
                class="block focus:outline-none text-gray-100 hover:text-white"
                href="https://github.com/perfolio"
              >
                github.com
              </a>
            </div>
          </div>
        </div>
        <div
          class="pt-2 border-t border-carbon-700"
        >
          <div
            class="max-w-6xl mx-auto"
          >
            <div
              class="flex flex-col items-center px-10 py-2 text-sm md:flex-row"
            >
              <span
                class="text-white"
              >
                Â© Copyright 
                2020
                . All Rights Reserved.
              </span>
              <div
                class="flex flex-row md:flex-auto md:flex-row-reverse"
              >
                <a
                  href="https://www.netlify.com"
                >
                  <img
                    alt="Deploys by Netlify"
                    src="https://www.netlify.com/img/global/badges/netlify-light.svg"
                  />
                </a>
              </div>
            </div>
          </div>
        </div>
      </footer>
    </div>
  </div>
</body>
`;
